mona_sdk/client.py

```diff
@@ -52,61 +52,63 @@
 # Note: if RAISE_AUTHENTICATION_EXCEPTIONS = False and the client could not
 # authenticate, every function call will return false.
 # Use client.is_active() in order to check authentication status.
-RAISE_AUTHENTICATION_EXCEPTIONS = get_boolean_value_for_env_var(
+MONA_SDK_RAISE_AUTHENTICATION_EXCEPTIONS = get_boolean_value_for_env_var(
     "MONA_SDK_RAISE_AUTHENTICATION_EXCEPTIONS", False
 )
 
-RAISE_EXPORT_EXCEPTIONS = get_boolean_value_for_env_var(
+MONA_SDK_RAISE_EXPORT_EXCEPTIONS = get_boolean_value_for_env_var(
     "MONA_SDK_RAISE_EXPORT_EXCEPTIONS", False
 )
 
-RAISE_SERVICE_EXCEPTIONS = get_boolean_value_for_env_var(
+MONA_SDK_RAISE_SERVICE_EXCEPTIONS = get_boolean_value_for_env_var(
     "MONA_SDK_RAISE_SERVICE_EXCEPTIONS", False
 )
 
-SHOULD_USE_AUTHENTICATION = get_boolean_value_for_env_var(
+MONA_SDK_SHOULD_USE_AUTHENTICATION = get_boolean_value_for_env_var(
     "MONA_SDK_SHOULD_USE_AUTHENTICATION", True
 )
 
-SHOULD_USE_SSL = get_boolean_value_for_env_var("MONA_SDK_SHOULD_USE_SSL", True)
+MONA_SDK_SHOULD_USE_SSL = get_boolean_value_for_env_var("MONA_SDK_SHOULD_USE_SSL", True)
```

Comment:
Why are you doing all these changes? What are we gaining from using this prefix? There's a good reason to have it in the env var's name - but why do we need it in the internal rep?

Reply:
Easier to read/track when it has the same name all over the place

Reply:
Sorry, I disagree. While in many cases this makes sense, in this one - we have a prefix whose all point is to make sure we don't override other env vars the user has. But within the code, adding this prefix everywhere makes things less readable and more cumbersome.

Reply:
No worries, I'll change it back and will merge (formatter and some kwargs), ok?